perftools/stresstesting/login-and-learn.yml

```diff
@@ -0,0 +1,22 @@
+config:
```

Comment:
Going to leave it in the repo for now - but this conversation is not closed. Just want to get a release out!